packs/git/actions/clone.yaml

```diff
@@ -0,0 +1,25 @@
+---
+name: clone
+runner_type: run-remote
+description: "Clone a repository"
+enabled: true
+entry_point: ''
+parameters:
+  source:
```

Comment:
Yeah being able to provide location to a key or auth creds would really help. I agree this works for now.